src/pipeline/config.cpp

```diff
@@ -256,7 +256,7 @@ namespace librealsense
                 }
             }
 
-            return ctx->add_device(file)->create_device(false);
+            return ctx->add_device(file)->create_device();
```

Comment:
So playback device will get notifications now? Can it even be "removed"?

Reply:
No, playback device override the create_device() function and is not calling the function we change it's defaults.
So it's OK, nothing changed there.